bears/python/requirements/PySafetyBear.py

```diff
@@ -26,6 +29,18 @@ def cve_key_checker(vulnerability):
 Package = namedtuple('Package', ('key', 'version'))
 
 
+def _get_vulnerabilities(pkg, spec, db):
+    for entry in db[pkg]:
+        entry['cve'] = 'None' if entry['cve'] is None else entry['cve']
+        entry['id'] = entry['cve']
```

Comment:
you should only overwrite this if it is a cve.  If it isnt, you need to show the pyup vuln id.
find a solution.
many exist.
choose one.